packages/api/src/external/fhir/document/index.ts

```diff
@@ -163,22 +168,40 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
  * @param resource
  * @returns
  */
-function convertToFHIRResource(resource: Contained): Resource | undefined {
-  if (resource.resourceType === ResourceType.Patient && resource.id) {
+function convertToFHIRResource(resource: Contained, patientId: string): Resource | undefined {
+  if (resource.resourceType === "Patient" && resource.id && !resource.id.includes(patientId)) {
+    capture.message(`Found a matching Patient resource in CW docRef.contained`, {
+      extra: { context: `toFHIR.convertToFHIRResource`, resource, patientId, level: "warning" },
+    });
     return {
-      resourceType: ResourceType.Patient,
+      resourceType: "Patient",
       id: resource.id,
     };
-  } else if (resource.resourceType === ResourceType.Organization && resource.name) {
+  } else if (resource.resourceType === "Organization" && resource.name) {
     return {
-      resourceType: ResourceType.Organization,
+      resourceType: "Organization",
+      id: resource.id ?? undefined,
+      identifier: convertCWIdentifierToFHIR(resource.identifier),
       name: convertCWNameToString(resource.name),
+      address: convertCWAdressToFHIR(resource.address),
     };
-  } else if (resource.resourceType === ResourceType.Practitioner && resource.name) {
+  } else if (resource.resourceType === "Practitioner" && resource.name) {
     return {
-      resourceType: ResourceType.Practitioner,
+      resourceType: "Practitioner",
+      id: resource.id ?? undefined,
+      identifier: convertCWIdentifierToFHIR(resource.identifier),
       name: convertCWNameToHumanName(resource.name),
+      gender: convertCWGenderToFHIR(resource.gender?.coding),
     };
+  } else {
+    capture.message(`New Resource type on toFHIR conversion`, {
```

Comment:
so we don't forget why this is here should we be explicit with the message? perhaps:

`New Resource type on toFHIR conversion - might need to handle in CW doc ref mapping`